core/procedures.js

```diff
@@ -17,20 +17,25 @@
 goog.module('Blockly.Procedures');
 goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.Blocks');
-goog.require('Blockly.Events');
+/* eslint-disable-next-line no-unused-vars */
+const Abstract = goog.requireType('Blockly.Events.Abstract');
+/* eslint-disable-next-line no-unused-vars */
+const Block = goog.requireType('Blockly.Block');
+const Blocks = goog.require('Blockly.Blocks');
+const Events = goog.require('Blockly.Events');
+/* eslint-disable-next-line no-unused-vars */
+const Field = goog.requireType('Blockly.Field');
+const Msg = goog.require('Blockly.Msg');
+const Names = goog.require('Blockly.Names');
+const Variables = goog.require('Blockly.Variables');
+const Workspace = goog.require('Blockly.Workspace');
+/* eslint-disable-next-line no-unused-vars */
+const WorkspaceSvg = goog.requireType('Blockly.WorkspaceSvg');
+const Xml = goog.require('Blockly.Xml');
```

Comment:
`Xml` + `xml` seems like a recipe for confusion. How about `Xml` and `utilsXml` or `xmlUtils`?